Content.Server/SimpleStation14/Species/Shadowkin/Systems/ShadowkinSystem.cs

```diff
@@ -0,0 +1,158 @@
+using Content.Shared.Examine;
+using Content.Shared.IdentityManagement;
+using Content.Shared.SimpleStation14.Species.Shadowkin.Components;
+using Content.Shared.SimpleStation14.Species.Shadowkin.Events;
+using Content.Shared.SimpleStation14.Species.Shadowkin.Systems;
+using Robust.Shared.Random;
+
+namespace Content.Server.SimpleStation14.Species.Shadowkin.Systems
+{
+    public sealed class ShadowkinSystem : EntitySystem
+    {
+        [Dependency] private readonly ShadowkinPowerSystem _powerSystem = default!;
+        [Dependency] private readonly IRobustRandom _random = default!;
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<ShadowkinComponent, ExaminedEvent>(OnExamine);
+            SubscribeLocalEvent<ShadowkinComponent, ComponentInit>(OnInit);
+        }
+
+        private void OnExamine(EntityUid uid, ShadowkinComponent component, ExaminedEvent args)
+        {
+            if (args.IsInDetailsRange)
+            {
+                var powerType = _powerSystem.GetLevelName(component.PowerLevel);
+
+                if (args.Examined == args.Examiner)
+                {
+                    args.PushMarkup(Loc.GetString("shadowkin-power-examined-self",
+                        ("power", _powerSystem.GetLevelInt(component.PowerLevel)),
+                        ("powerMax", component.PowerLevelMax),
+                        ("powerType", powerType)
+                    ));
+                }
+                else
+                {
+                    args.PushMarkup(Loc.GetString("shadowkin-power-examined-other",
+                        ("target", Identity.Entity(uid, _entityManager)),
+                        ("powerType", powerType)
+                    ));
+                }
+            }
+        }
+
+        private void OnInit(EntityUid uid, ShadowkinComponent component, ComponentInit args)
+        {
+            if (component.PowerLevel <= ShadowkinComponent.PowerThresholds[ShadowkinPowerThreshold.Min] + 1f)
+                _powerSystem.SetPowerLevel(component.Owner, ShadowkinComponent.PowerThresholds[ShadowkinPowerThreshold.Okay]);
+
+            component.ForceSwapRate = _random.NextFloat(component.ForceSwapRateMin, component.ForceSwapRateMax);
+            component.TiredRate = _random.NextFloat(component.TiredRateMin, component.TiredRateMax);
+        }
+
+
+        public override void Update(float frameTime)
+        {
+            base.Update(frameTime);
+
+            var query = _entityManager.EntityQuery<ShadowkinComponent>(true);
```

Comment:
```suggestion
            var query = _entityManager.EntityQuery<ShadowkinComponent>();
```
Is there a reason you're including paused Shadowkin?